base_custom_info/models/custom_info.py

```diff
@@ -2,9 +2,10 @@
 # © 2015 Antiun Ingeniería S.L. - Sergio Teruel
 # © 2015 Antiun Ingeniería S.L. - Carlos Dauden
 # © 2015 Antiun Ingeniería S.L. - Jairo Llopis
-# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
 
 from openerp import api, fields, models
+from openerp.tools.safe_eval import safe_eval
 
 
 class CustomInfoModelLink(models.AbstractModel):
```

Comment:
Can we get these models in separate files please? [Relevant guideline](https://github.com/OCA/maintainer-tools/blame/master/CONTRIBUTING.md#L941)